packages/backend/src/graphql/resolvers/peer.ts

```diff
@@ -129,13 +129,40 @@ export const updatePeer: MutationResolvers<ApolloContext>['updatePeer'] =
 
 export const deletePeer: MutationResolvers<ApolloContext>['deletePeer'] =
   async (
-    parent,
+    _,
     args,
     ctx
   ): Promise<ResolversTypes['DeletePeerMutationResponse']> => {
-    // TODO:
-    console.log(ctx) // temporary to pass linting
-    return {}
+    const peerService = await ctx.container.use('peerService')
+    return peerService
+      .delete(args.id)
+      .then((peer: Peer | undefined) =>
```

Comment:
I'm not sure why the resolvers were written with the `.then/.catch` promise style, but keeping for consistency

Reply:
That is a very good question. We would have to ask @wilsonianb

Reply:
Looks like I changed it in
- https://github.com/interledger/rafiki/pull/929

I'm guessing I was inspired by the conciseness of some other resolver we had.